domain/model/state/modelstate.go

```diff
@@ -188,14 +199,19 @@ SELECT &dbUUID.uuid
 FROM model
 	`, uuid)
 	if err != nil {
-		return errors.Annotatef(err, "preparing check model exists statement")
+		return errors.Capture(err)
 	}
 
 	err = tx.Query(ctx, checkExistsStmt).Get(&uuid)
 	if err != nil && !errors.Is(err, sqlair.ErrNoRows) {
-		return internalerrors.Errorf("checking if model already exists: %w", err)
+		return errors.Errorf(
+			"creating readonly model information and checking if model already exists: %w",
```

Comment:
Same here, don't need the `creating readonly model information and` in every error